bundles/org.openhab.core.automation.module.script.rulesupport/src/test/java/org/openhab/core/automation/module/script/rulesupport/loader/AbstractScriptDependencyTrackerTest.java

```diff
@@ -40,15 +43,27 @@
 public class AbstractScriptDependencyTrackerTest {
 
     private static final String WATCH_DIR = "test";
+    private static final Path DEPENDENCY = Path.of("depFile");
+    private static final Path DEPENDENCY2 = Path.of("depFile2");
 
     private @NonNullByDefault({}) AbstractScriptDependencyTracker scriptDependencyTracker;
     private @Mock @NonNullByDefault({}) WatchService watchServiceMock;
+    private @NonNullByDefault({}) @TempDir Path rootWatchPath;
+    private @NonNullByDefault({}) Path depPath;
+    private @NonNullByDefault({}) Path depPath2;
 
     @BeforeEach
-    public void setup() {
-        when(watchServiceMock.getWatchPath()).thenReturn(Path.of(""));
+    public void setup() throws IOException {
+        when(watchServiceMock.getWatchPath()).thenReturn(rootWatchPath);
         scriptDependencyTracker = new AbstractScriptDependencyTracker(watchServiceMock, WATCH_DIR) {
         };
+
+        depPath =rootWatchPath.resolve(WATCH_DIR).resolve(DEPENDENCY);
+        depPath2 =rootWatchPath.resolve(WATCH_DIR).resolve(DEPENDENCY2);
```

Comment:
```suggestion
        depPath = rootWatchPath.resolve(WATCH_DIR).resolve(DEPENDENCY);
        depPath2 = rootWatchPath.resolve(WATCH_DIR).resolve(DEPENDENCY2);
```

Reply:
Interesting. Why did spotless not fix that?